### PR TITLE
feat: add outputs

### PR DESCRIPTION
**Pull request by BouvetMorten**
Adding following outputs to Service Bus Namespace: 

- Namespace ID
- Namespace Endpoint

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -19,6 +19,18 @@ variable "sku" {
   default     = "Basic"
 }
 
+variable "system_assigned_identity_enabled" {
+  description = "Should the system-assigned identity be enabled for this Web App?"
```
Comment (hknutsen) [inline variables.tf]: Correct resource type:

```suggestion
  description = "Should the system-assigned identity be enabled for this Service Bus namespace?"
```

File: variables.tf [hunk 2]
```diff
@@ -19,6 +19,18 @@ variable "sku" {
...
+  type        = bool
+  default     = false
+}
+
+variable "identity_ids" {
+  description = "A list of IDs of managed identities to be assigned to this Web App."
```
Comment (hknutsen) [inline variables.tf]: Correct resource type:

```suggestion
  description = "A list of IDs of managed identities to be assigned to this Service Bus namespace."
```

**Review by hknutsen — CHANGES_REQUESTED** [review]
Added some comments regarding output descriptions.

What I noticed is that this PR adds variables for enabling system- and user-assigned identities, as well as corresponding outputs, however it doesn't actually implement the identity feature for the `azurerm_servicebus_namespace` resource.

File: variables.tf (hunk 2 above)
Comment (BouvetMorten) [reply variables.tf]: Good catch 👍

**Comment by BouvetMorten** [comment]
> Added some comments regarding output descriptions.
> 
> What I noticed is that this PR adds variables for enabling system- and user-assigned identities, as well as corresponding outputs, however it doesn't actually implement the identity feature for the `azurerm_servicebus_namespace` resource.

Should I maybe create a separate PR to implement the identity feature? Or rename this PR?

**Comment by hknutsen** [comment]
I think the cleanest approach would be for this PR to add the missing ID and endpoint outputs, then create a separate PR for identity implementation.

**Review by hknutsen — APPROVED** [review]

**Pull request merged**